privacyidea/api/lib/prepolicy.py

```diff
@@ -2106,22 +2106,28 @@ def required_piv_attestation(request, action=None):
                                                  REQUIRE_ACTIONS.REQUIRE_AND_VERIFY in list(require_att)
 
 
-def check_excluded_tokeninfo_fields(request=None, action=None):
-    user_object = request.User
-    role = g.logged_in_user.get("role")
-    if role == ROLE.USER:
-        return True
-
-    pols = Match.user(g, scope=SCOPE.USER, action=ACTION.TOKENINFO_API_EXCLUDE, user_object=user_object)\
-        .action_values(unique=True)
-
-    # when there is a policy
-    if pols:
-        # get the excluded fields
-        action_string = list(pols)[0]
-        # create a list
-        action_list = action_string.split(';')
-        # set the fields as a list to the request-data
-        request.all_data['excluded_tokeninfo_fields'] = action_list if len(action_list) > 0 else None
+def check_hide_tokeninfo(request=None, action=None):
+    # user_object = request.User
+    # role = g.logged_in_user.get("role")
+    # if role == ROLE.USER:
+    #     return True
+    #
+    # pols = Match.user(g, scope=SCOPE.USER, action=ACTION.TOKENINFO_API_EXCLUDE, user_object=user_object)\
+    #     .action_values(unique=True)
+    #
+    # # when there is a policy
+    # if pols:
+    #     # get the excluded fields
+    #     action_string = list(pols)[0]
+    #     # create a list
+    #     action_list = action_string.split(';')
+    #     # set the fields as a list to the request-data
+    #     request.all_data['excluded_tokeninfo_fields'] = action_list if len(action_list) > 0 else None
+
```

Comment:
Please rename `hidden_tokeninfo_fields` to `hidden_tokeninfo`.
Please delete the commented lines and add a short description of the decorator function in """ quotes.

Maybe something like:

``` python
"""
This decorator checks for the policy `hide_tokeninfo` and sets the 
`hidden_tokeninfo` parameter. The given tokeninfo keys will be
removed from the response.

Check ACTION.HIDE_TOKENINFO

The decorator wraps GET /token/

:param request: The request that is intercepted during the API call
:type request: Request Object
:param action: An optional Action
:type action: basestring
:returns: Always true. Modified the parameter request
"""
```